app/views/employments/AddEmploymentPayrollNumberFormView.scala.html

```diff
@@ -14,64 +14,87 @@
-            legend=Messages("tai.addEmployment.employmentPayrollNumber.legend"),
-            legendHint=None,
-            ('_groupDivClass, "form-group" ),
-            ('_radioOption1WrapAttribute, "data-target=payrollNumberEntry-container"),
-            ('_groupClass, "inline"),
-            ('_fieldsetAttributes, "id=payrollNumberChoice" ),
-            ('_legendSpanClass, "visually-hidden")
-        )}
+                @govukRadios(Radios(
+                    errorMessage = payrollNumberForm.errors.filter(_.key == "payrollNumberChoice").asTextErrorMessages.headOption,
+                    idPrefix = Some("payrollNumberChoice"),
+                    name = "payrollNumberChoice",
+                    items = Seq(
+                        RadioItem(
+                            content = Text(messages("tai.label.yes")),
+                            value = Some("Yes"),
+                            checked = payrollNumberForm.data.get(AddEmploymentPayrollNumberForm.PayrollNumberChoice).exists(_ == "Yes"),
+                            conditionalHtml = Some(
+                                govukInput(Input(
+                                    id = "payrollNumberEntry",
+                                    name = "payrollNumberEntry",
+                                    value = payrollNumberForm.data.get(AddEmploymentPayrollNumberForm.PayrollNumberEntry),
+                                    label = Label(
+                                        classes = "govuk-label--s",
+                                        content = Text(messages("tai.addEmployment.employmentPayrollNumber.label")),
+                                    ),
+                                    errorMessage = payrollNumberForm.errors.filter(_.key == "payrollNumberEntry").asTextErrorMessages.headOption
+                                ))
+                            )
+                        ),
+                        RadioItem(
+                            content = Text(messages("tai.label.no")),
+                            value = Some("No"),
+                            checked = payrollNumberForm.data.get(AddEmploymentPayrollNumberForm.PayrollNumberChoice).exists(_ == "No")
+                        )
+                    )
+                ).withFormField(payrollNumberForm(AddEmploymentPayrollNumberForm.PayrollNumberChoice)))
 
-        <div id="payrollNumberEntry-container" class="@if(payrollNumberForm(AddEmploymentPayrollNumberForm.PayrollNumberChoice).value.getOrElse("") != AddEmploymentPayrollNumberForm.YesValue){js-hidden} data-payrollNumberEntry">
-            @input(
-                field = payrollNumberForm(AddEmploymentPayrollNumberForm.PayrollNumberEntry),
-                labelText = Messages("tai.addEmployment.employmentPayrollNumber.label"),
-                classes = Seq("form-control input--cleared"),
-                panel = true
-            )
+                @govukButton(Button(content = Text(messages("tai.continue"))))
```

Comment:
does this need to say submit like it did before?

Reply:
so in the messages file: tai.continue == tai.submit  == "Continue"
imo being that both of them point to continue I'd use continue but again buttons are submitted... I'm happy either way